lib/assets/javascripts/cartodb/table/map/map_options_dropdown.js

```diff
@@ -124,7 +124,7 @@ cdb.admin.MapOptionsDropdown = cdb.ui.common.Dropdown.extend({
       if (!_.contains(self.collection.pluck("type"), overlay_type)) {
         self.mapOptions.set(overlay_type, false);
       }
-    })
+    });
```

Comment:
While these code style changes are generally good, it doesn't seem to be enforced by cartodb, so maybe we should avoid these types of changes in code unrelated to the changes we're making to reduce the chance of merge conflicts?